README.md

```diff
@@ -2,37 +2,108 @@
 
 A simple Docker & Drupal for development. This is all.
 
-### Apache + PHP 5.6
+## Nginx + PHP 5.6
 
-This branch is still on progress. For more stable, use the nginx branches.
+This branch is still in progress.
 
 What's included:
 
-  - PHP 5.6 (based on apache)
-  - Mariadb
+- Nginx
+- PHP-FPM 5.6
+- Mariadb
 
 Check the other branches of this repo for some more options.
 
 ### Official containers only
+
 All of this is based on officials containers, so you won't download any
 specific containers to use this project.
 
 ### Get started
+
 All your Drupal files should be placed into the folder "web".
 
+```sh
+mkdir web
+```
+
+Add the project domain to your local Hosts file, and point it to
```

Comment:
Could we make this optional? If yes, let's move it bellow as "Optional". Optional domain name ... etc.
I want people to be able to start quickly, then if they want more, they would check the README part "Optional".